### PR TITLE
Add ingest_radial_velocity function. 

### PR DESCRIPTION
**Pull request by kelle**
Utility function and test to ingest radial velocities.

### CODE REVIEW

**Review by Will-Cooper — DISMISSED** [review]
Could we please get some docstrings and type hints in the `ingest_gaiadr3` script? Is it true that whatever is the most recent upload is what we are taking as *adopted*? Can we also get a test of NaN or inf data, hopefully those should be rejected.

**Comment by kelle** [comment]
The `ingest_gaiadr3` script was just copied and pasted from a stale PR.  It's in the `scripts\` directory, not the utils directory. I think it can receive more time and attention once it's actually used as part of an ingest. But right now, I just didn't want to lose the work.  So, long story short, I don't want to spend time tidying that script.

The function worthy of more scrutiny is the `ingest_radial_velocity` function in the `astrometry` module.

**Comment by kelle** [comment]
I will add type hints to the ingest_radial_velocity script and something about units. 

Also, since there is not yet *any* rv data in SIMPLE, I will also modify the schema to add units to the column names.

**Comment by kelle** [comment]
Because I've improved the testing setup in #471, I should really wait for it to be merged before progressing with this one.

File: .github/workflows/gen-db.yml [hunk 1]
```diff
@@ -9,7 +9,6 @@ on:
   workflow_dispatch:  # manual execution
   release:
     types: [published]
-    branches: [main]
```
Comment (dr-rodriguez) [inline .github/workflows/gen-db.yml:12]: Curious why this change? I guess we sometimes want to release from non-main branches?

File: simple/utils/astrometry.py [hunk 2]
```diff
@@ -357,3 +357,181 @@ def ingest_proper_motions(
...
     return
+
+
+def ingest_radial_velocity(
+    db: Database,
+    *,
+    source: str = None,
+    rv: Union[Quantity, float] = None,
+    rv_err: Optional[Union[float, Quantity]] = None,
+    reference: str = None,
```
Comment (dr-rodriguez) [inline simple/utils/astrometry.py:368]: I think the types here (and the docstring descriptions) are incorrect: these are not strings/floats/Quantities, but lists of string/floats/Quantities (or numpy arrays/astropy Tables or similar)

Comment (dr-rodriguez) [reply simple/utils/astrometry.py:368]: Actually never mind, I think I confused `ingest_radial_velocity` with `ingest_radial_velocities`

Comment (dr-rodriguez) [reply simple/utils/astrometry.py:368]: I find it interesting (and confusing) that this function takes a single value while the others in this module take lists of values- we should try to be consistent if possible.

File: scripts/ingests/Gaia/ingest_gaiadr3.py [hunk 3]
```diff
@@ -0,0 +1,89 @@
+from scripts.ingests.ingest_utils import *
+from scripts.ingests.utils import *
```
Comment (dr-rodriguez) [inline scripts/ingests/Gaia/ingest_gaiadr3.py:2]: May be good to explicitly list which functions we are using from here rather than importing *

File: scripts/ingests/Gaia/ingest_gaiadr3.py [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+
+# update_ref_tables()
+
+
+def add_gaia_rvs(data, ref):
+    unmasked_rvs = np.logical_not(data["radial_velocity"].mask).nonzero()
+    rvs = data[unmasked_rvs]["db_names", "radial_velocity", "radial_velocity_error"]
+    refs = [ref] * len(rvs)
+    ingest_radial_velocities(
```
Comment (dr-rodriguez) [inline scripts/ingests/Gaia/ingest_gaiadr3.py:63]: This tripped me up because astrometry.py is in this PR and refers to a different singular function. Should we aim to rename these as `ingest_multiple_radial_velocities` or similar?

File: simple/utils/astrometry.py [hunk 5]
```diff
@@ -357,3 +357,181 @@ def ingest_proper_motions(
...
+    -------
+    flags: dict
+        'added' : bool
+        'skipped' : bool
+
+    Examples
+    ----------
+    > ingest_radial_velocity(db, my_source, rv=my_rv, rv_err=my_rv_unc,
+                             reference=my_rv_ref,
+                             raise_error = False)
```
Comment (dr-rodriguez) [inline simple/utils/astrometry.py:400]: Example may be better with some actual numbers/strings as opposed to variable names to demonstrate the types of inputs (similar to what's in the tests).

File: simple/utils/astrometry.py (hunk 2 above)
Comment (Will-Cooper) [reply simple/utils/astrometry.py:368]: I still hold that all functions in utils should be built to be single entry ingests but with like a `np.vectorize` decorator

**Review by dr-rodriguez — APPROVED** [review]
